java.base/src/main/java/java/net/NetUtil.java

```diff
@@ -47,14 +49,14 @@
 @Tracking("src/java.base/unix/native/libnet/SocketImpl.c")
 @Tracking("src/java.base/windows/native/libnet/net_util_md.c")
 @Tracking("src/java.base/windows/native/libnet/SocketImpl.c")
-class NetUtil {
+public class NetUtil {
```

Comment:
This would be the first (I think) time we've made a non-public class public in a public JDK package. I think this might set a bad precedent though. In other cases we've used the patcher to create an accessor in the other package for bypassing access controls; that might be better to do here. WDYT?

Reply:
Makes sense.  I'll adjust.